src/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+</head>
+<body>
+    <div id="container">
+        <header id="header">
+            <h2>Today's To Do</h2>
+        </header>
+        <hr>
+        <div id="input">
+            <input id="inputToDo" placeholder="Add to your list..." type="text">
+        </div>
+        <hr>
+        <div id="toDos">
+        </div>
+        <footer>
+            <a href="#">Clear all completed</a>
+        </footer>
+
+    </div>
+</body>
```

Comment:
- [Optional] Consider using an indentation of two spaces in all your documents. This is a convention and also best practice.